index.js

```diff
@@ -190,6 +187,9 @@ app.post("/resetPassword", function(req, res) {
 						console.log(e + "\n\n\n ***CHECK YOUR FIREWALL FOR PORT 587***");
 					}
 				});
+			//res.send("No User...<script>alert('user Email does not exist'); window.location = 'forgot'</script>");
+		} else {
+			res.json({ error: "User not found in our database.", redirectUrl: "forgot" });
```

Comment:
doesn't hurt, but if we've got an error we should omit redirectUrl or set it to null as the client won't redirect but just display the error